docs/react-testing-library/example-intro.md

```diff
@@ -118,7 +162,39 @@ const greetingTextNode = await waitForElement(() =>
 )
 ```
 
-### Assert
+#### Assert
+
+```jsx
+// fetch.js
```

Comment:
Should this be above "Arrange"? It seems odd to include it in the middle here with "Assert"